css-brochure-form/styles.css

```diff
@@ -0,0 +1,63 @@
+form {
+  width: 50%;
+  margin: auto;
+  margin-left: 6.8in;
+}
+
+p {
+  padding-left: 10px;
+  margin-bottom: 6px;
+  font-family: BrandonGrotesque, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+input {
+  margin: 4px;
+  width: 53%;
+  padding: 10px;
+  border-style: solid;
+  border: 1px solid rgb(226 226 226);
+  border-radius: 5px;
+  font-size: 17px;
+  font-family: AtlasGrotesk, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  flex
+}
+
+h1 {
+  text-align: center;
+  color: rgb(211 89 71);
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: bold;
+  font-weight: 300;
+  font-size: 20px;
+}
+
+select {
+  margin: fixed;
+  width: 458px;
+  height: 47px;
+  background-color: rgb(240 240 240);
+  border-style: solid;
+  border: 2px solid rgb(226 226 226);
+  border-radius: 5px;
+  font-size: 13px;
+}
+
+button {
+  margin: auto;
+  background-color: rgb(219 83 65);
+  font-size: 16px;
+  color: white;
+  text-align: right;
```

Comment:
I doubt you want to align the text inside the button to the right. You likely intended to apply this to the `div` that holds the `button` as a child.

Reply:
You can minimize my comment, but it still needs to be addressed 👀

Reply:
Sorry! I didn't mean to minimize it, I couldn't read the entire code because I thought it was in the way 😅